src/rgw/rgw_lc.cc

```diff
@@ -820,7 +820,8 @@ static inline bool worker_should_stop(time_t stop_at, bool once)
   return !once && stop_at < time(nullptr);
 }
 
-int RGWLC::handle_multipart_expiration(rgw::sal::Bucket* target,
+int RGWLC::handle_multipart_expiration(const DoutPrefixProvider *dpp,
```

Comment:
Same here, the dpp here is not needed, it should be a `this` since `RGWLC` is a `DoutPrefixProvider`.